identity_iota/Cargo.toml

```diff
@@ -16,6 +16,7 @@ description = "Framework for Self-Sovereign Identity with IOTA DID."
 # identity_comm = { version = "=0.5.0-dev.4", path = "../identity_comm", optional = true }
 identity_account = { version = "=0.6.0", path = "../identity_account", default-features = false, optional = true }
 identity_account_storage = { version = "=0.6.0", path = "../identity_account_storage", default-features = false, optional = true }
+identity_agent = { version = "=0.6.0", path = "../identity_agent", default-features = false, optional = true }
```

Comment:
nit: 
This implicitly creates a feature of the name `identity_agent` which we don't really want (at least not at this point).  By introducing my suggestion on line 65 (available since Rust 1.60) this can be avoided.